### DISCUSSION TITLE
Custom Aspect Ratios etc

### DISCUSSION THREAD

**Discussion by spazestudios** [NovaoTech/solaris#33] [Ideas]
so Ive been thinking in order to make this better than scratch / something people will use we need some stuff and some stuff that ive seen on other scratch type things that are really useful are : custom aspect ratios the ability to convert to html/zip/exe etc. 
also the profile should have more combustibility I always thought 

idk just stuff that would be cool

**Comment by icarusinthewings** [comment]
Because our editor and project loader will be through turbowarp, this is 100% possible!
